src/main/java/io/supertokens/webserver/WebserverAPI.java

```diff
@@ -49,10 +50,11 @@ public abstract class WebserverAPI extends HttpServlet {
         supportedVersions.add("2.15");
         supportedVersions.add("2.16");
         supportedVersions.add("2.17");
+        supportedVersions.add("2.18");
```

Comment:
- change json file, changelog
- change plugin interface version, json for that too